content/en/catalyst/fund9/challenges/grow-east-asia-grow-cardano/campaign-brief.md

```diff
@@ -0,0 +1,53 @@
+
+The nature of the proposals for this challenge is open-ended. Their only requirement is to originate or focus on East Asia and include east Asian team members or focus on East Asian participants.
+
+For instance, the overwhelming majority of Stake Pools reside in North America and Europe. The Cardano network will be more resilient when Stake Pools are evenly distributed across the globe. The Catalyst community needs to step up and intentionally make that happen.
+
+Cardano started life in East Asia (Japan and Hong Kong), and Emurgo focuses on East Asia. Let's build upon that.
+
+#### What this is:
+
+A call to action; Activate the Cardano community's ability to recognise, reward the opportunity to make the world a better place. East Asia is fertile ground for the adoption of Cardano. The return on intention (ROI) is likely to be high because projects in young and developing economies face less economic inertia.
+
+Addressing different parts of the world directly and intentionally through funding challenges makes our community more accessible. It makes us more resilient. It helps us learn and understand the needs of communities that make up our ecosystem.
+
+It is a vision that needs votes to become an achievement. When the challenge succeeds, the beneficiary is the whole Cardano community. It is a call to close our prejudices and open our eyes, hearts and minds to good people who lack the benefits many of us were born with—allowing others with fewer privileged circumstances to rise and succeed.
+
+### Why is it important?
+The Cardano community leans heavily toward North America and Europe. Growth elsewhere requires our support and initiative as a community.
+
+### What does success look like?
+A broad spectrum of people across Asia and connected to Asia, coming together and forming teams to grow our ecosystem.
+
+### Key Metrics to measure
+At the end of this challenge, we will be asking ourselves some questions about the quality of participation :
+
+- The number of proposals originating from East Asian participants in Project Catalyst
+- The quantity and quality of funded proposals originating from East Asian teams
+- The observed impact of funded projects (meeting their KPIs)
+- Did we increase the participation of the Cardano community and Catalyst teams with East Asian residents?
+- Was there an increase in Daily/Monthly Active Wallets (DAW/MAW) as a proxy measure of momentum and engagement
+- Did we increase the number of Stake Pools and Stake Pool Operators based in East Asia?
+
+We will also be asking some specific questions about the quality of the proposals :
+
+- Is there good feedback on funded project progress, failure or success that help us understand the nuances of deploying Cardano solutions across East Asia?
+What is the novelty and appropriateness of solutions?
```

Comment:
Add (-) in front of this sentence, 
'- What is the novelty and appropriateness of solutions?'